main.py

```diff
@@ -37,13 +38,14 @@
         'max_grid_export': 13,
         'penalty_import': 0.01,
         'penalty_export': 0.01,
-        'manage_voltage': True,
+        'manage_voltage': False,
         'penalty_overvoltage': 0.1,
         'penalty_undervoltage': 0.1,
         'max_voltage': 1.001,
         'min_voltage': 0.999,
         'weight_network_costs': 1,
-        'subset_line_losses_modelled': 30
+        'subset_line_losses_modelled': 1,
+        'compare_pandapower_optimisation': False
     }
 }
```

Comment:
maybe it is best if we don't commit the main files if only the settings change? then it disturbs the settings of whatever experiment we were running. I will create an issue to have the settings be a separate file that we add on .gitignore.